Automattic/wp-calypso
# SMP: add loading placeholder

#### Proposed Changes

* Create new package/component `LoadingPlaceholder`
* Add component stories to storybook
* Add loading placeholders while fetching for the sites
* Add animation delay to the rows to improve the design

![66203_dynamic-rows-delay](https://user-images.githubusercontent.com/779993/183120778-ce17c64a-bad3-4a72-8470-de5eae2515e4.gif)

#### Screencast


https://user-images.githubusercontent.com/779993/182966595-9ff3645f-2f28-4a92-ba81-ba38f5c6103a.mp4



#### Testing Instructions

* Go to `/sites-dashboard`
* Open Chrome Devtools > Application > IndexedDB > and delete it
*  Refresh the page
* Observe the loading placeholders are visible for a brief moment until the sites are loaded.

#### Pre-merge Checklist






Closes #66203
